doc/api/net.md

```diff
@@ -902,6 +905,19 @@ For TCP connections, available `options` are:
   **Default:** `false`.
 * `keepAliveInitialDelay` {number} If set to a positive number, it sets the initial delay before
   the first keepalive probe is sent on an idle socket.**Default:** `0`.
+* `autoSelectFamily` {boolean|number}: If set to `true` or to a positive number, it enables
+  a family autodetection algorithm that loosely implements section 5 of [RFC 8305][]. The `all`
+  option passed to lookup is set to `true` and the sockets attempts to connect to all
+  returned AAAA and A records, in sequence, until a successful connection is established.
+  The first returned AAAA record is tried first, then the first returned A record and so on.
+  The option specifies how much time in milliseconds to wait for a attempt to finish before trying
```

Comment:
I don't think `finish` is the right word here.

```suggestion
  The option specifies how much time in milliseconds to wait for an attempt to finish before trying
```

Reply:
👍